R/whatWQPdata.R

```diff
@@ -135,6 +135,22 @@ whatWQPdata <- function(..., saveFile = tempfile()){
   
   y[,grep("Count",names(y))] <- sapply(y[,grep("Count",names(y))], as.numeric)
   
+  names(y)[names(y) == "type"] <- paste("type",letters[1:length(names(y)[names(y) == "type"])],sep="_")
```

Comment:
@wdwatkins This PR has a ton of changes...but it's mostly just getting the siteNumbers arg consistent, and *not* using siteNumbers as a variable name for examples. The only meat of it is here. I'm checking if the geojson is all "Point"'s, then converting the coordinate list column to regular'ol numeric lat/lon. Jim said currently there's only Points, and no future though of doing anything other than points. So, this should be pretty good for awhile. Also, even if they do add something that's not just a point, this code won't get triggered, so I still think it would be OK. Let me know if you're going to have time to have a look here in the near future, or if you'd like me to find someone else.